web-components/src/components/cards/ProjectCard/ProjectCard.tsx

```diff
@@ -1,6 +1,8 @@
 import { useState } from 'react';
 import { SxProps, Theme, useTheme } from '@mui/material';
 import clsx from 'clsx';
+import InfoTooltip from 'web-components/src/components/tooltip/InfoTooltip';
+import { CREATE_POST_DISABLED_TOOLTIP } from 'web-marketplace/src/components/organisms/SellOrdersActionsBar/SellOrdersActionsBar.constants';
```

Comment:
do not import variables from web-marketplace into web-components
web-marketplace depends on web-components and it should not be the other way around
also as I had explained, since we set up translation, it's best to provide any displayed strings as props in web-components components.